test/verify/check-machines

```diff
@@ -41,6 +43,23 @@ def getNetworkDevice(m):
 
     return device
 
+def readFile(name):
+    content = ''
+    if os.path.exists(name):
+        with open(name, 'r') as f:
+            content = f.read().replace('\n', '')
+    return content
+
+def createUser(machine, user_group):
+    user_name = 'test_' + user_group + '_' + randomString()
```

Comment:
Let's please not make this random, this makes comparing logs and debugging tests unnecessarily hard. This never gets called more than once, so just appending a `_user` or just a `1` should do.

Reply:
Applied the suggestion ;)